packages/Telephony/Actions/BatchTerminationCharacterInput.cs

```diff
@@ -22,6 +22,7 @@ public class BatchTerminationCharacterInput : RegexAggregatorInput
         public const string Kind = "Microsoft.Telephony.BatchTerminationCharacterInput";
 
         private const string _dtmfCharacterRegex = @"^[\d#\*]+$";
```

Comment:
Looks like we cannot change the termination character from #. TerminationConditionRegexPattern and _terminationCharacter are not used anywhere.

Reply:
It actually works with any termination character (in the DTMF character set). I did some tricky bits with the property where it sets the parents property as part of the getter/setter (kind of a virtual property pattern).

Reply:
Agreed. I misread the _dtmfCharacterRegex = @"^[\d#\*]+$" as something else. Def. worth adding more comments for the slow regex readers.